examples/basic/android/app/build.gradle

```diff
@@ -127,9 +131,10 @@ android {
 }
 
 dependencies {
-    compile "com.android.support:appcompat-v7:25.2.0"
-    compile "com.facebook.react:react-native:+"  // From node_modules
-    compile project(':react-native-video')
+    implementation project(':react-native-video')
+    compile "com.android.support:appcompat-v7:28.0.0"
+    implementation "com.facebook.react:react-native:+"  // From node_modules
+    // implementation project(':react-native-video')
```

Comment:
dead code

Reply:
in the example folder. i have just changed from "25.2.0" to "28.0.0"